src/tests/ftest/util/launch_utils.py

```diff
@@ -1233,6 +1294,12 @@ def run_tests(self, logger, result, repeat, slurm_setup, sparse, fail_fast, stop
                 # Stop logging to the test log file
                 logger.removeHandler(test_file_handler)
 
+        # Cleanup any specified files at the end of testing
+        for file, info in cleanup_files.items():
+            command = command_as_user(f"rm -fr {file}", info['user'])
+            if not run_remote(logger, info['hosts'], command).passed:
+                return_code |= 16
```

Comment:
Looks like this is only used for systemctl now, but IMO this is a good candidate to group files together if possible. I.e.
```
rm -fr <file1> <file2> [...]
```

Reply:
That suggestion is easily handled by the adjusting the `cleanup_files` input.  The key could be a space separated list of files as a string.